### DISCUSSION TITLE
how do you install this on coffee linux?

### DISCUSSION THREAD

**Discussion by DOOMMARINE117** [Schneegans/Burn-My-Windows#423] [Q&A]
Hi, im trying to install this theme on coffee linux. which is a arch based distro. 

im using linux mint layout and etc. 

im tryong to install this theme. will this work? im installing it from kde store. 

if not, how to uninstall it?

**Comment by Schneegans** [comment]
Hi there! I have never heard of Coffee Linux before. You will need to explain in more detail what desktop environment this distro uses. Do you use GNOME? If so, which version? Do you use KDE? If so, which version of KWin? Or does it come with a completely different desktop environment?
